languageserver/server/server.go

```diff
@@ -189,8 +189,8 @@ type Server struct {
 	accessCheckMode               sema.AccessCheckMode
 	// reportCrashes decides when the crash is detected should it be reported
 	reportCrashes bool
-	// memberAccountAccessHandler stored to apply when building per-project configs
-	memberAccountAccessHandler sema.MemberAccountAccessHandlerFunc
+	// memberAccountAccessHandler is the optional function that is used to determine access for accounts
+	memberAccountAccessHandler func(projectID string, checker *sema.Checker, memberLocation common.Location) bool
```

Comment:
Maybe define a type alias for the function type to make this field and `WithMemberAccountAccessHandler` more readable